app/views/invite_requests/show.js.erb

```diff
@@ -1,5 +1,7 @@
 <% if @invite_request %>
   $j("#invite-status").html("<%= escape_javascript(render "invite_requests/invite_request", invite_request: @invite_request) %>");
+<% elsif @invitation %>
+  $j("#invite-status").html("<%= escape_javascript(render "invitation", invitation: @invitation) %>");
 <% else %>
-  $j("#invite-status").html("<p>Sorry, we can't find the email address you entered. If you had used it to join our invitation queue, it's possible that your invitation may have already been emailed to you; please check your spam folder, as your spam filters may have placed it there.</p>");
+  $j("#invite-status").html("<p>Sorry, we can't find the email address you entered.</p>");
```

Comment:
We have a little system using Rails and HTML data-* attributes for some things where we toggle text (e.g., the Favorite Tag button), but I think the closest we have to translated text in a .js.erb file is https://github.com/otwcode/otwarchive/blob/d87652399df73fa5e3b829b0089529501a381538/app/views/kudos/index.js.erb#L2C

What if we move the text to the partial, like:
```
<% if @invite_request %>
  $j("#invite-status").html("<%= escape_javascript(render "invite_requests/invite_request", invite_request: @invite_request) %>");
<% elsif @invitation %>
  $j("#invite-status").html("<%= escape_javascript(render "invitation", invitation: @invitation) %>");
<% else %>
  $j("#invite-status").html("<%= escape_javascript(render "invitation", invitation: nil) %>");
<% end %>
```

And then have the invitation partial do something like: 
```
<% if invitation.blank? %>
  <p><%= t(".email_not_found")</p>
<% else %>
  <!--Descriptive page name, messages and instructions-->
  <h2 class="heading" id="invite-heading">
    <%= t(".title", email: invitation.invitee_email) %>
  </h2>
  etc
 ```

 Or it could be its own partial, even, to avoid the giant if/else.